README.md

```diff
@@ -20,11 +20,74 @@ module "gce-ilb" {
 }
 ```
 
+## Testing
+
+This module comes packaged with tests with the goal of exercising the module in
+various forms. In order to exercise them, you need the following:
+
+1. A project in Google Cloud Platform.
+2. An IAM user with credentials downloaded locally and set with the
+  `GOOGLE_APPLICATION_CREDENTIALS` environment variable.
+3. The following roles attached to that IAM user:
+    ```text
+    roles/compute.networkAdmin
+    roles/compute.loadBalancerAdmin
+    ```
+4. A workstation with `docker` installed and the daemon running.
+
+To run the tests, build a docker container using the following command:
```

Comment:
These instructions are no longer accurate since the Dockerfiles have been removed.

Reply:
Ahh this was one change I'd made on the spinnaker effort that I hadn't ported back. Good catch!